extensions/amp-ad-network-adsense-impl/0.1/responsive-state.js

```diff
@@ -149,6 +186,27 @@ export class ResponsiveState {
     return /** @type {!ResponsiveState} */ (state);
   }
 
+  /**
+   * Convert the element to container width responsive.
+   *
+   * @param {!Element} element
+   * @return {!ResponsiveState} responsive state
+   * @private
+   */
+  static fallBackToContainerWidth_(element) {
+    const width = String(element.parentElement.clientWidth);
```

Comment:
@jridgewell what's the recommended way to get a parents' width with the least repercussions? We are trying to resize the ad to the parents size in certain cases.

Reply:
Checked with Justin.

Might have to add comments `/*OK*/` to avoid the complains.